packages/apps/proof-of-us/src/EditorForm/EditorForm.tsx

```diff
@@ -83,6 +83,7 @@ export const EditorForm: FC<IProps> = ({ signer, onClose }) => {
           name="label"
           defaultValue={signer?.name}
           maxLength={35}
+          className={mobileInputClass}
```

Comment:
just add this to the textfield component.
then we have the same style for every textfield